specs/Cryptography/Ed25519.md

```diff
@@ -0,0 +1,69 @@
+# Ed25519
+
+Reference: https://tools.ietf.org/html/rfc8032#section-5.1
+
+## Public key
+```rust
+pub struct ED25519PublicKey(pub [u8; ed25519_dalek::PUBLIC_KEY_LENGTH]);
```

Comment:
Still too much Rust. Also, there is no good reason to reference an API of a third-party library. Try to imagine someone with zero Rust knowledge reading this, will he understand anything in this line?